independent-publisher-connectors/Companies House/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2615 @@
+    "/company/%7B%7BCompanyNumber%7D%7D/persons-with-significant-control/individual/%7B%7BPCSId%7D%7D": {},
+    "/company/%7B%7BCompanyNumber%7D%7D/persons-with-significant-control-statements/%7B%7BStatementId%7D%7D": {},
+    "/company/%7B%7BCompanyNumber%7D%7D/uk-establishments": {},
+    "//officers/%7B%7BOfficerId%7D%7D/appointments": {},
+    "/company/%7B%7BCompanyNumber%7D%7D/filing-history/%7B%7BTransactionId%7D%7D": {},
+    "/company/%7B%7BCompanyNumber%7D%7D/charges": {},
+    "/company/%7B%7BCompanyNumber%7D%7D/charges/%7B%7BChargeId%7D%7D": {},
+    "/company/%7B%7BCompanyNumber%7D%7D/registers": {},
+    "/company/%7B%7BCompanyNumber%7D%7D/registered-office-address": {},
+    "/company/%7B%7BCompanyNumber%7D%7D/officers": {},
+    "/company/%7B%7BCompanyNumber%7D%7D/appointments/%7B%7BAppointmentId%7D%7D": {},
+    "/company/{CompanyNumber}": {
+      "get": {
+        "summary": "Find Company By Number",
+        "description": "Find a Company By their Company Number",
+        "operationId": "CompanyByNumber",
+        "parameters": [
+          {
+            "name": "CompanyNumber",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Specify the Company Number",
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Company Number"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "jurisdiction": {
+                  "type": "string",
+                  "description": "jurisdiction"
+                },
+                "has_been_liquidated": {
+                  "type": "boolean",
+                  "description": "has_been_liquidated"
+                },
+                "registered_office_address": {
+                  "type": "object",
+                  "properties": {
+                    "postal_code": {
+                      "type": "string",
+                      "description": "postal_code"
+                    },
+                    "address_line_1": {
+                      "type": "string",
+                      "description": "address_line_1"
+                    },
+                    "region": {
+                      "type": "string",
+                      "description": "region"
+                    },
+                    "country": {
+                      "type": "string",
+                      "description": "country"
+                    },
+                    "address_line_2": {
+                      "type": "string",
+                      "description": "address_line_2"
+                    },
+                    "locality": {
+                      "type": "string",
+                      "description": "locality"
+                    }
+                  },
+                  "description": "registered_office_address"

```

Comment:
@MattCollins-Jones **All the description values of properties in the swagger** are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
@sriyen-msft I see the requirements for these now for not allowing certain characters. I have hopefully updated them all, apologies if I've missed any or if there are any additional details I need to add in.